app/views/miq_request/_prov_field.html.haml

```diff
@@ -48,6 +48,16 @@
     .form-group
       %label.col-md-2.control-label{:valign => "top"}
         = field_hash[:description]
+        - if field_hash[:help]
+          %a{:tabindex => 0,
+             :role             => "button",
+             :"data-toggle"    => "popover",
+             :"data-trigger"   => "focus",
+             :"data-html"      => "true",
+             :title            => "",
+             :"data-content"   => "#{field_hash[:help]}",
+             :"data-placement" => "top"}
+            %span.fa.fa-info-circle
```

Comment:
Great, thanks! TBH using `object_id` feels a bit hacky, but this being a generic partial, I don't think we can prevent it .. so, :+1: :)

(And actually, while bootstrap's popover does handle reinitialization cleanly, patternfly's wrapper does not, so.. :))

So **yay** from me :)

> If it's fine with you I'll squash the commits together and remove the WIP tag.

Just a note though, please note that ManageIQ policy is *not* to squash commits, unless it's purely debugging things like "Add console.log", "Remove console.log"...